Formula/votca.rb

```diff
@@ -0,0 +1,54 @@
+class Votca < Formula
+  desc "Versatile Object-oriented Toolkit for Coarse-graining Applications"
+  homepage "https://www.votca.org/"
+  url "https://github.com/votca/votca/archive/refs/tags/v2023-rc.1.tar.gz"
+  sha256 "b992d94b92dd501a32f2378609577aa94f4af06e529431859a082b5390add137"
+  license "Apache-2.0"
+
+  bottle do
+    sha256 arm64_ventura:  "84cad302d2074c53943d6a30e8febd32c78cb6dc2d0b3f613a36b0eb7e00ab22"
+    sha256 arm64_monterey: "9cd72621866b05e4f54f3de0cf842f6ac5b1fcde09151a2d54d7c64600f82335"
+    sha256 arm64_big_sur:  "2f6e26d5987d7c0e5dd33fa54bd2c671da79abb5089530713ac7d2d6086bce18"
+    sha256 ventura:        "811c68a3c5efe07c2fd8eb9ee034f9dc5d770d6f8b1da2a7e4e530c63d266eea"
+    sha256 monterey:       "b01407fcdb3ac2357561fa60c6b87d5ac954b8bae4990c90017001a3bb7e6c17"
+    sha256 big_sur:        "62ca47b5ed4d6773aa72d40a8e4b86a384a005803d037f220957d6e5fb957623"
+    sha256 x86_64_linux:   "eff58dc88ee3c6bffdab6bf26ca347da5e80240192a970d229af69969691123d"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "eigen"
+  depends_on "expat"
+  depends_on "fftw"
+  depends_on "gcc" # for OpenMP
+  depends_on "gromacs"
+  depends_on "hdf5"
+
+  on_macos do
+    depends_on "libomp"
+  end
```

Comment:
This should be removed, right? Per https://github.com/Homebrew/homebrew-core/pull/129648#issuecomment-1553839689, `libgomp` will be provided through `gcc`?

```suggestion
```

Reply:
Well, if you want to build with clang, you will still need this block.